.github/workflows/generate-preview-link.yml

```diff
@@ -112,22 +139,38 @@ jobs:
                     exit 1
                   fi
             - name: 'Generate preview link comment'
+              if: always()
               id: generate_preview_url
               uses: actions/github-script@v3
               with:
                   github-token: ${{ github.token }}
                   script: |
-                      const preview_url = "${{steps.publish-to-pages.outputs.preview_url}}"
-                      const comment = [
-                          `**Preview Link**: ${preview_url}`,
-                          '| Name | Result |',
-                          '| :--- | :------ |',
-                          `| **Build status**  | Completed ✅ |`,
-                          `| **Preview URL**  | [Visit Preview](${preview_url}) |`,
-                          ''
-                        ].join('\n')
+                      let comment;
+                      const action_url = "${{github.server_url}}/${{github.repository}}/actions/runs/${{github.run_id}}"
+                      const status = "${{steps.publish-to-pages.outcome}}"
```

Comment:
If the build step fails should we also show this comment?

Reply:
Imo maybe we can separate the steps, like:
```suggestion
      - name: 'Generate failure comment'
        if: failure()
        ....generate fail comment
        
      - name: 'Generate preview link comment'
         if: success()
         ...generate success comment
        
```

Reply:
<img width="832" alt="Screenshot 2023-09-27 at 1 00 48 PM" src="https://github.com/binary-com/deriv-com/assets/103016120/43e2eb95-e744-4dfd-9141-d254178e0b68">

Reply:
Done. Thanks for the review @adrienne-deriv 🙏